docs/book/src/topics/managedcluster.md

```diff
@@ -265,6 +265,36 @@ spec:
   ...
 ```
 
+### Disable Local Accounts in AKS when using Azure Active Directory
+
+When deploying an AKS cluster, local accounts are enabled by default. 
+Even when you enable RBAC or Azure AD integration,
+--admin access still exists as a non-auditable backdoor option.
+Disabling local accounts closes the backdoor access to the cluster
+Example to disable local accounts for AAD enabled cluster.
+
+```yaml
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: AzureManagedMachinePool
+metadata:
+  ...
+spec:
+  aadProfile:
+    managed: true
+    adminGroupObjectIDs:
+    -  00000000-0000-0000-0000-000000000000 # group object id created in azure.
+  disableLocalAccounts: true  
+  ...
+```
+
+Note: CAPZ and CAPI requires access to the target cluster to maintain and manage the cluster. 
+Disabling local accounts will cut off direct access to the target cluster. 
+CAPZ and CAPI can access target cluster only via the Service Principal, 
+hence the user has to provide appropriate access to the Service Principal to access the target cluster. 
+User can do that by adding the Service Principal to the appropriate group defined in Azure and 
```

Comment:
I'm struggling to set this up. Is there a link to Azure docs that describe how to do this?

Reply:
Can we get into a call sometime today ? I can explain how this works.

Reply:
Synced with @LochanRn and it looks like I need to create a group with a `type` of "Security" to be able to add a Service Principal as a member and I don't seem to have permission to do that.

@CecileRobertMichon Is there a chance I could be granted that permission to test this? Not a huge deal if not since @LochanRn agreed to help test this on my ASO PR but if I could test myself that would be a lot easier.